core/core.go

```diff
@@ -186,6 +188,27 @@ func New(config Config) (*App, error) {
 	}
 	config = unquoteConfig(config)
 
+	// Ensure ETHEREUM_RPC_MAX_REQUESTS_PER_24_HR_UTC is reasonably set given BLOCK_POLLING_INTERVAL
+	var per24HrPollingRequests int
+	if config.BlockPollingInterval > 1*time.Second {
```

Comment:
We should remove this if statement. The math is the same regardless of whether `config.BlockPollingInterval` is greater than 1 second. The way it's currently written means that a duration of 1.5 seconds is truncated to 1 second which yields incorrect results. I think the math is also more complicated than it needs to be.

How about this instead?

```go
per24HrPollingRequests = int((24 * time.Hour) / config.BlockPollingInterval)
```

Reply:
The issue with your proposal is that it overflows time.Duration (see: https://play.golang.org/p/VaAauCNj1i4. I've gone ahead and removed the conditional statement. We now always compute from perSecond which works.

Reply:
My solution doesn't overflow `time.Duration`. You just needed to add parentheses in your example. See https://play.golang.org/p/nDSxy5hAt5O

Reply:
Ah! Amazing. Ok, let's do this then.